src/schema/annotation.yaml

```diff
@@ -81,48 +81,13 @@ classes:
     is_a: FunctionalAnnotationTerm
     description: >-
       A pathway is a sequence of steps/reactions carried out by an organism or community of organisms
-    slots:
-      - has_part
-    slot_usage:
-      has_part:
-        range: Reaction
-        required: true
-        description: >-
-          A pathway can be broken down to a series of reaction step
     id_prefixes:
       - KEGG_PATHWAY
       - COG
     exact_mappings:
       - biolink:Pathway
-
-  Reaction:
-    class_uri: "nmdc:Reaction"
-    is_a: FunctionalAnnotationTerm
-    description: >-
-      An individual biochemical transformation carried out by a functional unit of an organism, 
-      in which a collection of substrates are transformed into a collection of products.
-      Can also represent transporters.
-    slots:
-      - direction
-      - is_balanced
-      - is_diastereoselective
-      - is_fully_characterized
-      - is_stereo
-      - is_transport
-      - left_participants
-      - right_participants
-      - smarts_string
-    id_prefixes:
-      - KEGG.REACTION
-      - RHEA
-      - MetaCyc
-      - EC
-      - GO
-      - MetaNetX
-      - SEED
-      - RetroRules
-    exact_mappings:
-      - biolink:MolecularActivity
+    comments:
+      - might have reactions as their parts
```

Comment:
Should this be removed too?

Reply:
That's a comment on [Pathway](https://microbiomedata.github.io/nmdc-schema/Pathway/)

Reply:
It's been almost 2 months.. but I think I was saying we would remove the comment, because it no longer has reactions as their parts?

Reply:
I changed this to a `todos` with the following wording: 
> If we reverted to including Reaction in the schema, then a Reaction would be a reasonable part_of a Pathway